SonarExternal.cmake

```
@@ -791,6 +791,10 @@ function(build_glog)
   if(GLOG_VERSION VERSION_GREATER_EQUAL 0.5.0)
     set(libdir ${EXTERNAL_INSTALL_LIBDIR})
   endif()
+  set(libname glog)
+  if(CMAKE_BUILD_TYPE STREQUAL Debug)
+    string(APPEND libname d)
+  endif()
```

Not sure if this would be easier or harder to read, so I leave that up to someone else to decide, but here is an option
```suggestion
  set(glog_libname libglog)
  if(CMAKE_BUILD_TYPE STREQUAL Debug)
    string(APPEND glog_libname d)
  endif()
  string(APPEND glog_libname .a)
```